training-node/test/users.js

```diff
@@ -138,44 +180,46 @@ describe('users', () => {
         });
     });
   });
-  it('should fail auth because of missing property email', done => {
-    successUserCreate().then(res => {
+  describe('/users GET', () => {
+    it('should success search ', done => {
+      successUserCreate().then(res => {
+        successUserAuth().then(response => {
+          const token = response.headers[tokenManager.HEADER_NAME];
+
+          chai
+            .request(server)
+            .get('/users?offset=0&size=2')
+            .set(tokenManager.HEADER_NAME, token)
+            .then(result => {
+              dictum.chai(result);
```

Comment:
I'll do a test for checking that the pagination is working, counting the total users in the response should be ok

Reply:
awesome